src/styles/blocks/catalog.scss

```diff
@@ -7,6 +7,7 @@
   padding: 50px 0;
 
   &__card {
+    transition: transform 200ms, color 300ms;
```

Comment:
Do you really need to have different transition duration for different properties?
I think it is better to keep it the same for the whole page in order not to confuse the user)